store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -149,6 +150,10 @@
     private static final String IMAP_DELETE_RIGHTS = "xted";
     private static final String IMAP_ADMIN_RIGHTS  = "a";
 
+    private static Map<String, Set<String>> processedDeletes =
```

Comment:
Do we sticky for IMAP or will this only work if the request hits this same service? (i.e. single node)

Reply:
This is classic it always goes to the same node. Fixed map